Example/Example for Delirium/Modules/Dynamic Type/FontFamilyHeader.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  FontFamilyHeader.swift
+//  Delirium
+//
+//  Created by Nikki Vergracht on 31/05/2017.
+//  Copyright © 2017 CocoaPods. All rights reserved.
+//
+
+import UIKit
+
+class FontFamilyHeader: UITableViewHeaderFooterView {
+    
+    let label = UILabel()
+    
+    class var reuseIdentifier: String {
+        return String(describing: self)
+    }
+    
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.backgroundColor = .clear
+        label.textAlignment = .center
+        if #available(iOS 8.2, tvOS 8.2,  *) {
```

Comment:
If you want you may change the deployment target of the Example project(s). This way you can remove this check.